lib/compiler/src/translator/environ.rs

```diff
@@ -89,6 +89,9 @@ impl<'data> ModuleEnvironment<'data> {
         assert!(self.module_translation_state.is_none());
         let module_translation_state = translate_module(data, &mut self)?;
         self.module_translation_state = Some(module_translation_state);
+
+        self.module.hash = Some(ModuleHash::xxhash(data));
```

Comment:
We chatted about this, how are we solving the issue for not computing the hash always? This is not addressed